src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] agrs) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        truck.doWork();
```

Comment:
create a Machine array with Truck, Bulldozer, and Excavator and call methods doWork() and stopWork() in a loop.